<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createroom</class>
 <widget class="QDialog" name="createroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="create_lobby_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Lobby</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_easy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_medium">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_hard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
  <widget class="QLabel" name="diff_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Difficulty:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="num_players_cb">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>30</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 players</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 players</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="num_players_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>140</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of players:</string>
   </property>
  </widget>
  <widget class="QLabel" name="err_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>135</y>
     <width>250</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
